roles/install-deps/tasks/main.yml

```diff
@@ -24,6 +24,13 @@
   tags:
     - install
 
+- name: Centos 7.3 nodejs Hack (bug 1481008 / 1481470)
+  when: ansible_distribution == "CentOS" and ansible_distribution_major_version == "7"
```

Comment:
If this is going to be fixed in 7.4, can you also add a check for the minor version <= 3 ?

Reply:
Yeah, agree, would be better to have a check on the minor version as well.

Reply:
Since it was a temporary fix and 7.4 was not released, I thinked it was ok. Furthermore ansible_distribution_minor does not exists.
But I have found an another way to test it. I will fix it

Reply:
Fair enough.